libs/navigation/navigation.js

```diff
@@ -4,7 +4,7 @@ const blockConfig = [
     name: 'global-navigation',
     targetEl: 'header',
     appendType: 'prepend',
-    params: ['imsClientId', 'searchEnabled', 'unav', 'customLinks', 'jarvis'],
+    params: ['imsClientId', 'searchEnabled', 'unav', 'customLinks', 'jarvis', 'layout'],
```

Comment:
You don't need to pass the param layout since it is not used in the block code through milo configs.

Reply:
Done removed from params.